frontend/src/views/datenbank/WorkDetail.view.vue

```diff
@@ -0,0 +1,161 @@
+<script setup lang="ts">
+import RisViewLayout from "@/components/RisViewLayout.vue"
+import RisEmptyState from "@/components/RisEmptyState.vue"
+import { ref, computed } from "vue"
+import type { HeaderBreadcrumb } from "@/components/RisHeader.vue"
+import { useGetNormWork, useGetNormExpressions } from "@/services/normService"
+import { useNormWorkEliPathParameter } from "@/composables/useNormWorkEliPathParameter"
+import Splitter from "primevue/splitter"
+import SplitterPanel from "primevue/splitterpanel"
+import { NormExpressionEli } from "@/lib/eli/NormExpressionEli"
+import { formatDate } from "@/lib/dateTime"
+import { useElementId } from "@/composables/useElementId"
+import { DokumentExpressionEli } from "@/lib/eli/DokumentExpressionEli"
+import type { TreeNode } from "primevue/treenode"
+import { Tree } from "primevue"
+import { RouterLink, useRoute } from "vue-router"
+import { useDokumentExpressionEliPathParameter } from "@/composables/useDokumentExpressionEliPathParameter"
+
+const workEli = useNormWorkEliPathParameter()
+const { expressionsHeadingId } = useElementId()
+const route = useRoute()
+
+const {
+  data: normWork,
+  isFetching: isFetchingNormWork,
+  error: normWorkError,
+} = useGetNormWork(workEli)
+
+const {
+  data: normExpressions,
+  isFetching: isFetchingNormExpressions,
+  error: normExpressionsError,
+} = useGetNormExpressions(workEli)
+
+const isLoading = computed(() => {
+  return isFetchingNormWork.value || isFetchingNormExpressions.value
+})
+
+const breadcrumbs = computed<HeaderBreadcrumb[]>(() => {
+  const base = [
+    {
+      key: "normWorkTitle",
+      title: () => normWork.value?.title ?? "...",
+    },
+  ]
+
+  if (route.name === "DatenbankWorkExpressionDetail") {
+    const expressionEli = useDokumentExpressionEliPathParameter("expression")
+    console.log(expressionEli)
+    return [
+      ...base,
+      {
+        key: "expressionInforceDate",
+        title: () =>
+          formatDate(
+            DokumentExpressionEli.fromString(
+              expressionEli.value?.toString() ?? "",
+            ).pointInTime,
+          ),
+      },
+    ]
+  }
+  return base
+})
+
+function getInForceDateFromEli(eli: string) {
+  try {
+    return formatDate(NormExpressionEli.fromString(eli).pointInTime)
+  } catch {
```

Comment:
when does this error case happen? And should we log something when it happens?

Reply:
when the ELI string that doesn't match the expected format, malformed, missing parts but i have now inlined the function within the computed treeNodesExpressions.